pandas/core/series.py

```diff
@@ -918,12 +918,49 @@ def _set_values(self, key, value):
     @deprecate_kwarg(old_arg_name='reps', new_arg_name='repeats')
     def repeat(self, repeats, *args, **kwargs):
         """
-        Repeat elements of an Series. Refer to `numpy.ndarray.repeat`
-        for more information about the `repeats` argument.
+        Repeat elements of a Series.
+
+        Each element of the Series is repeated 'repeats' times.
+
+        Parameters
+        ----------
+        repeats : int
+            The number of repetitions for each element.
+        *args
+            These parameters will be passed to a validation function.
+        **kwargs
+            These parameters will be passed to a validation function.
 
         See also
         --------
-        numpy.ndarray.repeat
+        Series.append : Concatenate two or more Series.
+        pandas.concat : Concatenate two or more DataFrames.
+        numpy.repeat : Repeat elements of an array.
+
+        Returns
+        -------
+        Series
+            The repeated version of the Series.
+
+        Examples
+        --------
+        >>> df = pd.DataFrame({'col1' : ['A','B','C'],'col2' : [ 0 , 1 , 2 ]})
+        >>> df
+        col1  col2
```

Comment:
Spacing looks off. Should move it over a few spaces I think. Double check that this is right.